### DISCUSSION TITLE
v19.10.7-3.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#108] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.10.7`  
Current YTLite Version: `3.0.1`  
Current App Compatibility: `iOS/iPadOS 14.0` or later

**RELEASE 19.10.7-3.0.1:** (2024-03-23)

- Updated Youtube from `19.10.5` to `19.10.7`
- Updated submodules
  
### Issues in this Release:
- On some device iSponsorBlock crash, disable it to fix the crash
- LowContrastMode does not work *(Compatible only with YouTube v16.46.5-v17.38.10)*
- VersionSpoofer (Lite) doesn't work with YTLite *(as the version is already spoofed by YTLite to fix YTClassicVideoQuality)*
- Hiding Shorts creates an empty space instead #55

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.10.7-3.0.1'>v19.10.7-3.0.1 - YTLitePlus</a>.</em>

**Comment by huandney** [comment]
Hi, is there any place where we can get the decrypted YouTube ipa to do the Building?

**Reply by Balackburn** [reply]
https://armconverter.com/decryptedappstore/us
https://decrypt.day/
https://anyipa.me/
